vendor/modules.txt

```diff
@@ -192,7 +192,7 @@ github.com/spf13/cobra
 # github.com/spf13/pflag v1.0.5
 ## explicit
 github.com/spf13/pflag
-# github.com/stretchr/testify v1.4.0
+# github.com/stretchr/testify v1.6.1
```

Comment:
@pratikjagrut any specific reason we're upgrading package in this PR ?

Reply:
got it

Reply:
Resolving this conversation.